chart/Chart.yaml

```
@@ -46,24 +46,22 @@ annotations:
     - name: Documentation
       url: https://airflow.apache.org/docs/helm-chart/1.17.0/
   artifacthub.io/screenshots: |
+    - title: Home Page
+      url: https://airflow.apache.org/docs/apache-airflow/stable/_images/home_dark.png
+    - title: DAG Overview Dashboard
+      url: https://airflow.apache.org/docs/apache-airflow/stable/_images/dag_overview_dashboard.png
     - title: DAGs View
-      url: https://airflow.apache.org/docs/apache-airflow/3.0.0/_images/dags.png
-    - title: Datasets View
-      url: https://airflow.apache.org/docs/apache-airflow/3.0.0/_images/datasets.png
+      url: https://airflow.apache.org/docs/apache-airflow/3.0.2/_images/dags.png
+    - title: Assets View
+      url: https://airflow.apache.org/docs/apache-airflow/stable/_images/asset_view.png
```

Why stable vs 3.0.2?
```suggestion
      url: https://airflow.apache.org/docs/apache-airflow/3.0.2/_images/asset_view.png
```

---

copy/paste link when I was verifying the image links were working or not

---

fixed in https://github.com/apache/airflow/pull/51594/commits/6b71e5553390a73074c4991c697496b8656b3415